beacon-chain/forkchoice/protoarray/proposer_boost_test.go

```diff
@@ -163,7 +168,7 @@ func TestForkChoice_BoostProposerRoot_PreventsExAnteAttack(t *testing.T) {
 		// middle instead of the normal progression of (44 -> 34 -> 24).
 		require.Equal(t, f.store.nodes[1].weight, uint64(54))
 		require.Equal(t, f.store.nodes[2].weight, uint64(44))
-		require.Equal(t, f.store.nodes[3].weight, uint64(24))
+		require.Equal(t, f.store.nodes[3].weight, uint64(34))
```

Comment:
The test setup description does not match the test itself. Why has this changed?

Reply:
Hi @prestonvanloon my test assertions were actually wrong here! The reason is explained below. We were inserting a node at slot 2 when I was intending to insert at slot 3 instead